.github/workflows/preview.yml

```diff
@@ -24,7 +24,43 @@ jobs:
             ${{ runner.os }}-gems-
 
       - name: Build Jekyll site
-        uses: jerryjvl/jekyll-build-action@v1
+        uses: limjh16/jekyll-action-ts@v2
+        with:
+          enable_cache: true
+          ### Enables caching. Similar to https://github.com/actions/cache.
```

Comment:
Funny that comments are placed AFTER the option

Reply:
Yeah, that's how they did it in their example :).